src/main/webapp/permissions-manage-files.xhtml

```diff
@@ -227,7 +233,8 @@
                                     <h:outputText value="#{roleAssignment.assigneeDisplayInfo.title}"/> <h:outputText value=" (#{roleAssignment.assigneeDisplayInfo.affiliation})" rendered="#{!empty roleAssignment.assigneeDisplayInfo.affiliation}"/>
                                 </p:column>
                                 <p:column width="98%" headerText="#{bundle['dataverse.permissionsFiles.files.tabHeader.fileName']}" rendered="#{!empty manageFilePermissionsPage.selectedRoleAssignee}">
-                                    <h:outputText value="#{roleAssignment.definitionPoint.displayName}"/>
+                                    <h:outputText value=" (#{bundle['dataverse.permissionsFiles.files.deleted']}) " rendered="#{roleAssignment.deleted}"/>
```

Comment:
I think the "(Deleted)" text should be after the file name, not before, so you can easily scan the file names, since that is the value the table rows are sorted on. The same could be said about the file directory path that is added to this column (e.g "one/001.txt", "two/002.txt" "three/003.txt" shouldn't be the order in the table, if sorting by file path).

Reply:
Yup. moved Deleted to after File Name